test/api_java.jl

```diff
@@ -87,6 +172,9 @@
             i2 = cpo_java_intervalvar(model)
             @test typeof(cpo_java_intervalsequencevar(model, [i1, i2])) ==
                   IloIntervalSequenceVar
+
+            # Set parameters
+            set_parameters(model)
```

Comment:
Is it useful to test this here? (It's the only place where I wouldn't have called that function in this PR.)

Reply:
No it isn't. [Link](https://github.com/JuliaConstraints/CPLEXCP.jl/pull/10/commits/57e31025321f8fcc141b951c51cfd5c27c4676e4#diff-a96c425fa0553a595e9ce870fae44377a4d7077b474b08b3e6d7bd81162eb0e8L175).